composer.json

```diff
@@ -22,17 +22,26 @@
     },
     "autoload": {
         "psr-4" : {
-            "chobie\\" : "src"
+            "chobie\\" : "src/"
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "Tests\\chobie\\": "tests"
+            "Tests\\chobie\\": "tests/"
         }
     },
+	"scripts": {
+		"stan": "phpstan analyse",
+		"stan-setup": "cp composer.json composer.backup && composer require --dev phpstan/phpstan:^1.0.0 && mv composer.backup composer.json"
+	},
     "extra": {
         "branch-alias": {
             "dev-master": "2.0-dev"
         }
+    },
+    "config": {
+        "platform": {
+            "php": "7.3"
+        }
     }
```

Comment:
Please rollback.

Such an approach would prevent installing dependencies for other PHP versions.